presto-main/src/main/resources/webapp/src/components/ResourceGroupView.jsx

```diff
@@ -0,0 +1,349 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { addToHistory, formatDataSizeBytes, parseDataSize, truncateString } from "../utils";
+import { QueryListItem } from "./QueryList";
+
+const SPARKLINE_PROPERTIES = Object.freeze({
+    disableHiddenCheck: true,
+    fillColor: '#3F4552',
+    height: '75px',
+    lineColor: '#747F96',
+    spotColor: '#1EDCFF',
+    tooltipClassname: 'sparkline-tooltip',
```

Comment:
A small typo

```suggestion
    tooltipClassName: 'sparkline-tooltip',
```

Reply:
unfortunately, this is sparkline API, I  can't change it.